icubam/backoffice/templates/operational-dashboard.html

```diff
@@ -0,0 +1,85 @@
+{% extends "base.html" %}
+
+{% block links %}
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+<link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+<link rel="stylesheet" type="text/css" href="/static/dist/css/dashboard.css">
+
+{% end %}
+
+{% block content %}
+<div class="container operational-dash">
+    <div class="row text-center">
+        <h1 class='text-center'>Dashboard opérationel</h1>
+    </div>
+    <div class="row dashboard-filter">
+       <div class="dropdown">
+         <a class="btn btn-secondary dropdown-toggle" href="#" role="button" id="dropdownMenuLink" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+	   {{ current_region_name }}
+         </a>
+       
+         <div class="dropdown-menu" aria-labelledby="dropdownMenuLink">
+	   <a class="dropdown-item" onClick="return false;" href="">Toutes les régions</a>
+	   <div class="dropdown-divider"></div>
+	   {% for row in regions %}
+	   <a class="dropdown-item" onClick="return false;" href="?region={{ row['id'] }}">{{ row['name'] }}</a>
+	   {% end %}
+         </div>
+       </div>
+
+       <script>
+	$(".dropdown-menu .dropdown-item").click(function(e) {
+	   // reloads page with new parameters
+	   // TODO: use the bokeh server instead of hacks.
+           window.location.href = window.location.href.replace( /[\?#].*|$/, $(this).attr("href") ); 
+        });
+       </script>
+
+    </div>
+    <div class="row">
+        <div class="col-sm-5 dash-metrics">
+	   {% for row in metrics_layout %}
+	   <div class="row"> 
+	      {% for metric in row %}
+              <div class="col-sm-4">
+                 <div class="card">
+                    <div class="card-body">
+		       <h3 class="card-title">{{ metric['value'] }}</h3>
+		       <p class="card-text">{{ metric['label'] }}</p>
```

Comment:
Would you know a formatter for jinja2 files by any chance? Struggling with tabs here :/

Reply:
yeah. it s bit messy but no worries.